__tests__/app.test.js

```diff
@@ -39,13 +29,82 @@ describe("GET /api/topics ", () => {
     });
 });
 
-describe("GET /api/topic ", () => {
+describe("GET /api/topicz ", () => {
   test("when path error returns a 404 for bad path", () => {
     return request(app)
-      .get("/api/topic ")
+      .get("/api/topicz ")
       .expect(404)
       .then(({ body }) => {
         expect(body.msg).toBe("bad request sorry");
       });
   });
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles', () => { 
+  test(' should return status 200 and an array of article objects with the comment_count,', () => { 
+    return request(app)
+      .get('/api/articles')
+      .expect (200)
+      .then(({ body }) => {
+         const { articles } = body;
+        expect(articles).toHaveLength(12);
+        expect(articles).toBeInstanceOf(Array);
```

Comment:
Checking if articles is an array is not really necessary here, because you are using a forEach() array method, so the test would fail if articles wasn't an array.